Gruntfile.js

```diff
@@ -325,14 +267,11 @@ module.exports = function(grunt) {
   ]);
 
   grunt.registerTask("generateJS", [
-    "concat:plottable",
     "concat:svgtypewriter",
-    "concat:definitions",
     "sed:definitions",
```

Comment:
This will now get out of sync, as you no longer have the `build/` folder. Try deleting the `build/` folder and check what blows up.